kedro/templates/project/{{ cookiecutter.repo_name }}/conf/base/logging.yml

```diff
@@ -1,55 +1,44 @@
 version: 1
```

Comment:
Most of these changes are just formatting. What's actually changed here is:
* removed `json_formatter` which was never used anywhere
* removed the special case `kedro.x` loggers in favour of the generic `kedro` one, which did the same thing anyway
* altered `root` logger so it no longer shows all `INFO` level messages (it's now the same as the framework-side logging config and doesn't specify this explicitly, so defaults to Python's WARNING level)
* added a special logger for `{{ cookiecutter.python_package }}`. This is to ensure that calls [like this](https://github.com/kedro-org/kedro-starters/blob/main/spaceflights/%7B%7B%20cookiecutter.repo_name%20%7D%7D/src/%7B%7B%20cookiecutter.python_package%20%7D%7D/pipelines/data_science/nodes.py#L55) still emit logs